### PR TITLE
Expose cardsPerPage prop for OnCallList in OpsgeniePage

### PR DESCRIPTION
**Pull request by sblausten**
This exposes the new cardsPerPage prop for the OnCallList component at the top Page level so that plugin users can easily set it.

### CODE REVIEW

**Comment by martina-if** [comment]
@K-Phoen Hi Kevin! 👋 Can you take a look when you have a moment please? 😃

File: yarn.lock [hunk 1]
```diff
@@ -1081,63 +1081,63 @@
     "@babel/helper-validator-identifier" "^7.16.7"
     to-fast-properties "^2.0.0"
 
-"@backstage/app-defaults@^1.0.1":
```
Comment (K-Phoen) [inline yarn.lock:1084]: These changes probably don't belong in this PR :)

**Review by K-Phoen — COMMENTED** [review]
I'm in vacation and don't have a laptop with me, so I can't test this. But it looks good, except the yarn.lock file that looks like it contains unintented changes.
I suspect the CI will stop complaining once that's fixed and we should be good to go!

File: yarn.lock (hunk 1 above)
Comment (sblausten) [reply yarn.lock]: These are just the latest versions pulled in when running yarn install but I reverted the change.

**Comment by K-Phoen** [comment]
Merged! Thanks for the PR :)
